project_timesheet_time_control/views/project_project_view.xml

```diff
@@ -4,10 +4,6 @@
         <field name="name">Add timesheet time controls</field>
         <field name="model">project.project</field>
         <field name="inherit_id" ref="hr_timesheet.project_invoice_form" />
-        <field
-            name="groups_id"
-            eval="[(6, 0, [ref('hr_timesheet.group_hr_timesheet_user')])]"
```

Comment:
If you remove these groups from the view, you have to put them at field/page/group element level.